src/brogue/Recordings.c

```diff
@@ -1057,13 +1066,16 @@ void saveGame() {
         return; // Call me paranoid, but I'd rather it be impossible to embed malware in a recording.
     }
 
-    getAvailableFilePath(defaultPath, "Saved game", GAME_SUFFIX);
+    sprintf(defaultPath, "#%s depth %d%s",
+        abbreviateNumber(rogue.seed), rogue.depthLevel,
+        (rogue.wizard ? " (wizard)" : rogue.easyMode ? " (easy)" : ""));
```

Comment:
There is repeated code here, what do you think about factoring it into a `void defaultFileName(char *buf, ...)`?

Reply:
Will do!